frontendActions/src/main/java/com/virtuslab/gitmachete/frontend/actions/base/BaseResetBranchToRemoteAction.java

```diff
@@ -237,6 +240,10 @@ public void run(ProgressIndicator indicator) {
             log().error(result.getErrorOutputAsJoinedString());
             VcsNotifier.getInstance(project).notifyError(VCS_NOTIFIER_TITLE, result.getErrorOutputAsHtmlString());
           }
+
+          var repositoryRoot = getMainDirectory(gitRepository);
+          GitRepositoryManager.getInstance(project).updateRepository(repositoryRoot);
+          GitUtil.refreshVfsInRoot(repositoryRoot);
```

Comment:
Looks that this method isn't there in 201.x yet... and it looks it can be easily replaced with `VfsUtil.markDirtyAndRefresh`:

```
public static void refreshVfsInRoot(@NotNull VirtualFile root) {
    RefreshVFsSynchronously.trace("refresh root " + root);
    VfsUtil.markDirtyAndRefresh(false, true, false, root);
  }
```